### PR TITLE
2039 ai branding

### PR DESCRIPTION
**Pull request by corykon**
image

image

image

image

### CODE REVIEW

File: src/app/foundations/ai/ai-demo.component.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
+<div class="demo-content">
+    <h1>Healthcare.AI™ Branding</h1>
```
Comment (aaronbrethorst) [inline src/app/foundations/ai/ai-demo.component.html:2]: please replace all instances of ™ with `&#8482;`

File: src/app/foundations/ai/ai-demo.component.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    <h6>Last updated March 3, 2023</h6>
+
+    <hc-tile>
+        <h5 id="overview">Overview</h5>
+
+        <p>
+            <a target="_blank" href="https://healthcare.ai/">Heathcare.AI™</a> is HCAT's AI framework that expands the use and use cases for effective AI for our customers.
+
+        <p>In order to call out and tie together the uses of AI in our products, we recommend using standard visual branding.
+            When someone sees this branding we want them to think <span class="hc-font-em">"Oh! I know what this is, and it's going to be really useful. I love Healthcare.AI."</span>
```
Comment (aaronbrethorst) [inline src/app/foundations/ai/ai-demo.component.html:12]: Replace

`"Oh! I know what this is, and it's going to be really useful. I love Healthcare.AI."`

with

`&quot;Oh! I know what this is, and it's going to be really useful. I love Healthcare.AI.&quot;`

**Review by aaronbrethorst — COMMENTED** [review]
here are the HTML validation issues I noticed.

**Comment by corykon** [comment]
I'm getting an error in the build complaining about the version of ng-packager. This is being updated in the ng14 PR anyways, so I'm going to wait until that goes in, merge it back, see if the error resolves itself

**Review by andrew-frueh — APPROVED** [review]
All good on the  checks once we merged the Angular 14 branch. So I'll go ahead and merge this too.

**Pull request merged**